### DISCUSSION TITLE
Force Charge on Octopus Intelligent sessions

### DISCUSSION THREAD

**Discussion by gerd091986** [wills106/homeassistant-solax-modbus#587] [Q&A]
Hi There,

I pointed HA at the Pocketwifi adapter instead and got 256 entities back, working as intended. (I have setup the Modbus with a Waveshare RS485 to RJ45 adapter, but couldn't get this to work, only 4 entities shown.)

I am trying to edit the following values but they just change back:

- SolaX Manual Mode Control
- SolaX Manual Mode Select
- Solax Charger Use Mode

I can change them in the Solax App and this is reflected instantly in HA.

I have clicked the unlock and advanced unlock but nothing happens.

I am trying to force charge the battery when Octopus Intelligent sessions trigger. 

Any help would be very much appreciated on how to force charge from HA.

**Comment by neildhunter** [comment]
Is the inverter in idle mode? I've found that if it is then it won't take updates from HA...
Check what sensor.solax_run_mode is reporting in HA...

I had similar behaviour while trying to test some things out one evening with no PV and a discharged battery!

**Reply by gerd091986** [reply]
It's in Normal Mode, here are all the status currently:

![Solax Status](https://github.com/wills106/homeassistant-solax-modbus/assets/149315629/accaffb0-3795-476b-9ce3-aaa6c244e36c)
